bee-network/build.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::io::Result;
+
+fn main() -> Result<()> {
+    prost_build::compile_protos(
+        &[
+            "src/proto/handshake.proto",
+            "src/proto/packet.proto",
+            "src/proto/message.proto",
+        ],
+        &["src/"],
```

Comment:
What does this mean exactly ? Why src ?

Reply:
@Alex6323

Reply:
from the `prost-build` docs:
```
includes - Paths to directories in which to search for imports. Directories are searched in order. The .proto files passed in protos must be found in one of the provided include directories.
```

Reply:
But does it looks recursively then ? Because the protos are not in `src`. Why don't we pass `src/proto/` ?

Reply:
we could do that and be more specific, but:
1) we don't know if we want to keep the directory structure like that (`src` however will probably forever be correct)
2) we don't know if being more concrete here doesn't help with anything (compilation performance maybe?)
3) I've never seen anything else put there yet (but that doesn't mean anything)